flink-connectors/flink-sql-connector-hbase-2.2/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,35 @@
+flink-sql-connector-hbase
+Copyright 2014-2020 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- com.google.guava:guava:12.0.1
+- com.yammer.metrics:metrics-core:2.2.0
+- commons-codec:commons-codec:1.10
+- commons-configuration:commons-configuration:1.7
+- commons-lang:commons-lang:2.6
+- commons-logging:commons-logging:1.1.3
+- io.netty:netty-all:4.1.44.Final
+- org.apache.hbase:hbase-client:1.4.3
```

Comment:
The `hbase2 sql connector` has its own dependencies. You can reference `flink-sql-connector-hbase-2.2/target/classes/META-INF/DEPENDENCIES`

Reply:
I've referenced the DEPENDENCIES files both in 1.4 and 2.2, removing all the license related parts of NOTICE. Please confirm that it is fine like this.

Reply:
Sorry let you misunderstand this. the NOTICE file is ok for  1.4, and for  2.2, we need to create a NOTICE file which follows the Flink notice style[1],  I mean the content of NOTICE file could reference DEPENDENCIES.

[1]https://cwiki.apache.org/confluence/display/FLINK/Licensing#

Reply:
Thanks for clarifying, and doing this part. I've added your changes.